crates/smdk/src/generate.rs

```diff
@@ -1,14 +1,51 @@
+use anyhow::{Error, Result};
 use clap::Parser;
-use anyhow::Result;
+use cargo_generate::{GenerateArgs, TemplatePath, generate};
+
+const GIT_TEMPLATE: &str = "https://github.com/infinyon/fluvio-smartmodule-template";
```

Comment:
I think that we should use: https://github.com/infinyon/fluvio/tree/master/smartmodule/cargo_template instead

Reply:
Template should be inlined so there is no dependency to external file